src/app/mock-data.ts

```diff
@@ -1387,3 +1387,46 @@ export const fhir = {
   ]
 }
 };
+
```

Comment:
I think most of these are already in the spec file and should not be here since this file is going away.

Reply:
Fixed